extensions/cornerstone/src/utils/CornerstoneViewportDownloadForm.tsx

```diff
@@ -142,12 +142,16 @@ const CornerstoneViewportDownloadForm = ({
           const properties = viewport.getProperties();
 
           downloadViewport.setStack([imageId]).then(() => {
-            downloadViewport.setProperties(properties);
-
-            const newWidth = Math.min(width || image.width, MAX_TEXTURE_SIZE);
-            const newHeight = Math.min(height || image.height, MAX_TEXTURE_SIZE);
-
-            resolve({ width: newWidth, height: newHeight });
+            try {
```

Comment:
Something I changed cause an exception to happen 100%, probably something around the prefetch configuration/activation.

Reply:
How do you reproduce this problem?

Reply:
Run the integration tests with the other changes but not this one.